rust/integration-tests/src/sdk_tests/account_transaction_processor_tests.rs

```diff
@@ -0,0 +1,116 @@
+use ahash::AHashMap;
+use aptos_indexer_testing_framework::sdk_test_context::SdkTestContext;
+use sdk_processor::config::{
+    db_config::{DbConfig, PostgresConfig},
+    indexer_processor_config::IndexerProcessorConfig,
+    processor_config::{DefaultProcessorConfig, ProcessorConfig},
+};
+use std::collections::HashSet;
+
+pub async fn setup_acc_txn_processor_config(
+    test_context: &SdkTestContext,
+    staring_version: u64,
+    txn_count: usize,
+    db_url: &str,
+) -> (IndexerProcessorConfig, &'static str) {
+    let transaction_stream_config =
+        test_context.create_transaction_stream_config(staring_version, txn_count as u64); // since this will be always 1, we can remove from the arg list
+    let postgres_config = PostgresConfig {
+        connection_string: db_url.to_string(),
+        db_pool_size: 100,
+    };
+
+    let db_config = DbConfig::PostgresConfig(postgres_config);
+    let default_processor_config = DefaultProcessorConfig {
+        per_table_chunk_sizes: AHashMap::new(),
+        channel_size: 100,
+        deprecated_tables: HashSet::new(),
+    };
+
+    let processor_config = ProcessorConfig::AccountTransactionsProcessor(default_processor_config);
+    let processor_name = processor_config.name();
+    (
+        IndexerProcessorConfig {
+            processor_config,
+            transaction_stream_config,
+            db_config,
+            backfill_config: None,
+        },
+        processor_name,
+    )
+}
+
+#[allow(clippy::needless_return)]
+#[cfg(test)]
+mod tests {
+    use crate::{
+        diff_test_helper::account_transaction_processor::load_data,
+        sdk_tests::{
+            account_transaction_processor_tests::setup_acc_txn_processor_config,
+            run_processor_test, setup_test_environment, validate_json, DEFAULT_OUTPUT_FOLDER,
+        },
+    };
+    use aptos_indexer_test_transactions::IMPORTED_MAINNET_TXNS_145959468_ACCOUNT_TRANSACTION;
+    use aptos_indexer_testing_framework::{cli_parser::get_test_config, database::TestDatabase};
+    use sdk_processor::processors::account_transactions_processor::AccountTransactionsProcessor;
+    #[tokio::test(flavor = "multi_thread", worker_threads = 2)]
+    async fn mainnet_acc_txns_processor() {
```

Comment:
nit: add a new line after imports. could you also add doc string here?

Reply:
added! weird that the rust formatter doesn't do that